modules/widgets/top-posts.php

```diff
@@ -251,6 +251,8 @@ function widget( $args, $instance ) {
 				/** This filter is documented in modules/stats.php */
 				'gravatar_default' => apply_filters( 'jetpack_static_url', set_url_scheme( 'https://en.wordpress.com/i/logo/white-gray-80.png' ) ),
 				'avatar_size' => 40,
+				'width' => null,
+				'height' => null,
```

Comment:
Should add these to the docblock about `jetpack_top_posts_widget_image_options` below.